io.github.glaumar.QRookie.yml

```diff
@@ -0,0 +1,59 @@
+app-id: io.github.glaumar.QRookie
+runtime: org.kde.Platform
+runtime-version: "6.6"
+sdk: org.kde.Sdk
+command: qrookie
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=all
+  - --share=network
+  - --persist=.android
+modules:
+  - name: p7zip
+    buildsystem: simple
+    build-commands:
+      - install -pDm 0755 7za -t "$FLATPAK_DEST/bin/"
+    sources:
+      - type: archive
+        url: https://github.com/p7zip-project/p7zip/releases/download/v17.05/linux-cmake-p7zip.zip
+        sha256: ac5240801ef208b7e7018a353c0d699526f5a0367fb8be89ce89a7b2cafb7d4e
+
+  - name: android-tools
+    buildsystem: simple
+    build-commands:
+      - install -pDm 0755 adb -t "$FLATPAK_DEST/bin/"
+    sources:
+      - type: archive
+        url: https://dl.google.com/android/repository/platform-tools_r34.0.0-linux.zip
+        sha256: 8137c2834dea05cb64c1a8bc041ea00fcd43e3a8a29429ad4f25b8ee51efebf6
+
+  - name: qcoro
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/danvratil/qcoro/archive/refs/tags/v0.10.0.tar.gz
+        sha256: b7c8f00273ad27d85814bf4ec93eb6922c75656800a61d11854d36355a4a1aec
+
+  - name: librclone
+    buildsystem: simple
+    build-commands:
+      - install -pDm 0644 librclone.a -t "$FLATPAK_DEST/lib/"
+      - install -pDm 0644 librclone.h -t "$FLATPAK_DEST/include/"
+    sources:
+      - type: archive
+        url: https://github.com/glaumar/QRookie/releases/download/v0.1.3/librclone_x86_1.65.tar.gz
```

Comment:
why not building this from source?

Reply:
When I  try : flatpak-go-get-generator.py .flatpak-builder/build/librclone/ven
dor/modules.txt >  go-sources.json
I get error: 
 Failed to check https://goftp.io/server/v2: HTTP Error 403: Forbidden